common/tier4_state_rviz_plugin/src/autoware_state_panel.cpp

```diff
@@ -348,106 +501,120 @@ void AutowareStatePanel::onOperationMode(const OperationModeState::ConstSharedPt
+      state = Danger;
+      bgColor = QColor("#f08b8b");
       break;
 
     case RouteState::CHANGING:
-      text = "CHANGING";
-      style_sheet = "background-color: #FFFF00;";  // yellow
+      bgColor = QColor("#eef08b");
+      state = Pending;
       break;
 
     default:
-      text = "UNKNOWN";
-      style_sheet = "background-color: #FF0000;";  // red
+      state = None;
+      bgColor = QColor("#84c2e6");
       break;
   }
 
-  updateLabel(routing_label_ptr_, text, style_sheet);
+  routing_icon->updateStyle(state, bgColor);
 
   if (msg->state == RouteState::SET) {
     activateButton(clear_route_button_ptr_);
   } else {
+    clear_route_button_ptr_->setStyleSheet(
+      "QPushButton {"
+      "background-color: #292d30;color: #6e7276;"
+      "border: 2px solid #292d30;"
+      "font-weight: bold;"
+      "}");
     deactivateButton(clear_route_button_ptr_);
   }
 }
 
 void AutowareStatePanel::onLocalization(const LocalizationInitializationState::ConstSharedPtr msg)
 {
-  QString text = "";
-  QString style_sheet = "";
+  IconState state;
+  QColor bgColor;
+
   switch (msg->state) {
     case LocalizationInitializationState::UNINITIALIZED:
-      text = "UNINITIALIZED";
-      style_sheet = "background-color: #FFFF00;";  // yellow
+      state = None;
+      bgColor = QColor("#84c2e6");
       break;
 
-    case LocalizationInitializationState::INITIALIZING:
-      text = "INITIALIZING";
-      style_sheet = "background-color: #FFA500;";  // orange
+    case LocalizationInitializationState::INITIALIZED:
+      state = Active;
+      bgColor = QColor("#00e678");
       break;
 
-    case LocalizationInitializationState::INITIALIZED:
-      text = "INITIALIZED";
-      style_sheet = "background-color: #00FF00;";  // green
+    case LocalizationInitializationState::INITIALIZING:
+      state = Pending;
+      bgColor = QColor("#eef08b");
       break;
 
     default:
-      text = "UNKNOWN";
-      style_sheet = "background-color: #FF0000;";  // red
+      state = None;
+      bgColor = QColor("#84c2e6");
       break;
   }
 
-  updateLabel(localization_label_ptr_, text, style_sheet);
+  localization_icon->updateStyle(state, bgColor);
 }
 
 void AutowareStatePanel::onMotion(const MotionState::ConstSharedPtr msg)
 {
-  QString text = "";
-  QString style_sheet = "";
+  IconState state;
+  QColor bgColor;
+
   switch (msg->state) {
     case MotionState::STARTING:
-      text = "STARTING";
-      style_sheet = "background-color: #FFFF00;";  // yellow
+      state = Pending;
+      bgColor = QColor("#eef08b");
       break;
 
-    case MotionState::STOPPED:
-      text = "STOPPED";
-      style_sheet = "background-color: #FFA500;";  // orange
+    case MotionState::MOVING:
+      state = Active;
+      bgColor = QColor("#00e678");
       break;
 
-    case MotionState::MOVING:
-      text = "MOVING";
-      style_sheet = "background-color: #00FF00;";  // green
+    case MotionState::STOPPED:
+      state = Danger;
```

Comment:
```suggestion
      state = None;
```

Stop to have ➖ makes more sense.

❗ can be used for Unknown. (❓would be better though)